sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -29,8 +29,6 @@
     <PackageReference Include="System.Text.Json" />
     <PackageReference Include="System.Text.Encodings.Web" />
     <PackageReference Include="System.Memory.Data" />
-
-    <ExcludeFromProjectReferenceToConversion Include="System.ClientModel" />
```

Comment:
I think this was a remnant from very early alpha days due to large breaking changes.   Should have been removed a long time ago!